Content.Client/Forensics/ForensicScannerMenu.xaml.cs

```diff
@@ -52,6 +52,9 @@ public void UpdateState(ForensicScannerBoundUserInterfaceState msg)
             {
                 text.AppendLine(fiber);
             }
+            text.AppendLine();
+            text.AppendLine("DNA");
```

Comment:
use localization for this

Reply:
yea, already done it, gonna push it when i fix some stuff